demo/demo.py

```diff
@@ -8,7 +8,7 @@
 from nanodet.util import load_model_weight
 from nanodet.data.transform import Pipeline
 
-image_ext = ['.jpg', '.jpeg', '.webp', '.bmp', '.png']
+image_ext = ['.jpg', '.jpeg', '.webp', '.bmp', '.png','.JPG']
```

Comment:
I don't think .JPG is a necessary file extension. But if you think this is useful, other file extensions should also have a uppercase version.

Reply:
You can do something like this:
````python
image_ext = ['.jpg', '.jpeg', '.webp', '.bmp', '.png']
image_ext += [x.upper() for x in image_ext]
````